app/models/bulk_upload.rb

```diff
@@ -1,6 +1,7 @@
 class BulkUpload < ApplicationRecord
   enum log_type: { lettings: "lettings", sales: "sales" }
   enum rent_type_fix_status: { not_applied: "not_applied", applied: "applied", not_needed: "not_needed" }
+  enum failed: { blank_template: 1, wrong_template: 2 }
```

Comment:
Is there any benefit to having integers? Might be a bit more readable if we have something similar to log_type instead